stories/app/about-dialog.stories.tsx

```diff
@@ -2,9 +2,11 @@ import { FaMeteor } from 'react-icons/fa';
 
 import { AboutDialogToolbarButton } from '../../src/app/index';
 import { Toolbar } from '../../src/components/index';
+import { RootLayoutDecorator } from '../utils';
 
 export default {
   title: 'Components / AboutDialogToolbarButton',
+  decorators: [RootLayoutDecorator],
```

Comment:
The `RootLayoutDecorator` is required for all stories. Can you set it up to be a global decorator so that you don't have to pass it to every story?

https://storybook.js.org/docs/react/writing-stories/decorators#global-decorators